econml/sklearn_extensions/ensemble.py

```diff
@@ -523,21 +532,27 @@ def fit(self, X, y, sample_weight=None, sample_var=None):
                                                                        int(np.ceil(self.subsample_fr_ *
                                                                                    (X.shape[0] // 2))),
                                                                        replace=False)])
-            trees = Parallel(n_jobs=self.n_jobs, verbose=self.verbose,
-                             **_joblib_parallel_args(prefer='threads'))(
+            res = Parallel(n_jobs=self.n_jobs, verbose=self.verbose,
+                           **_joblib_parallel_args(prefer='threads'))(
                 delayed(_parallel_add_trees)(
                     t, self, X, y, sample_weight, s_inds[i], i, len(trees),
                     verbose=self.verbose)
                 for i, t in enumerate(trees))
+            trees = [t[0] for t in res]
+            numerators = [t[1] for t in res]
+            denominators = [t[2] for t in res]
```

Comment:
I think this is the same as:
```suggestion
            trees, numerators, denominators = zip(*res)
```
which is cleaner looking to me.